src/main/java/seedu/algobase/logic/parser/ParserUtil.java

```diff
@@ -55,6 +58,21 @@ public static Name parseName(String name) throws ParseException {
         return new Name(trimmedName);
     }
 
+    /**
+     * Parses a {@code String name} into a {@code PlanName}.
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @throws ParseException if the given {@code name} is invalid.
```

Comment:
I suppose it's not required to write thrown exception in JavaDoc for this project? (checkstyle doesn't complain on this anyways...)